baybe/utils/basic.py

```diff
@@ -54,6 +54,42 @@ def get_subclasses(cls: _C, recursive: bool = True, abstract: bool = False) -> l
     return subclasses
 
 
+def get_parent_classes(
+    cls: type,
+    recursive: bool = True,
+    abstract: bool = False,
+    include_class: bool = True,
+) -> list[type]:
+    """Return a list of parent classes for the given class.
+
+    Args:
+        cls: The base class to retrieve parent classes for.
+        recursive: If ``True``, indirect parent classes (i.e., parent classes of parent
+            classes) are included.
+        abstract: If ``True``, abstract parent classes are included.
+        include_class: If ``True``, the class itself is included.
```

Comment:
I ask this question once in a moonphase :D 
do we still need double ticks?

Reply:
Yes, double is still required ;)